subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaPluginExtension.java

```diff
@@ -129,17 +129,19 @@
     ModularitySpec getModularity();
 
     /**
-     * Configure the toolchain requirements for tasks that require a tool from the toolchain (e.g. {@link org.gradle.api.tasks.compile.JavaCompile}.
+     * Gets the project wide toolchain requirements that will be used for tasks requiring a tool from the toolchain (e.g. {@link org.gradle.api.tasks.compile.JavaCompile}).
+     * <p>
+     * Configuring a toolchain makes it illegal to set the {@code sourceCompatibility} or {@code targetCompatibility} on this extension.
```

Comment:
```suggestion
     * Configuring a toolchain cannot be used together with {@code sourceCompatibility} or {@code targetCompatibility} on this extension. Both values will be enforced by the toolchain.
```

Reply:
Replaced enforced with sourced in your suggestion.